tests/debugger/utils/probes/probe_snapshot_log_line.json

```diff
@@ -5,6 +5,9 @@
         "id": "",
         "version": 0,
         "captureSnapshot": true,
+        "capture": {
```

Comment:
capture object is optional and should not be a requirement for a probe definition

Reply:
As I wrote in the PR description, that test case should preferably be a separate system test. It hasn't been something that has been an issue since we launched the Node.js tracer, so I would say it's only optional in theory.

But I agree it would be good to support that, just not for this test, whose purpose is different.

Reply:
Capture object has well defined default values from the beginning, so to me it does not make sense to put this artifical capture object here

Reply:
`captureSnapshot` is required because it makes the difference between a regular log probe and a snapshot probe

Reply:
If we add a test for this scenario, the CI will be also broken for you. How it is different than adding this `@bug` ?

Reply:
Adding the `@bug` doesn't run this test and hence hides any issues related to what this test is testing. Also, it's incorrect calling it a bug as our backend is always adding the `capture` object to the probe payload sent via RC. I agree that we should probably support the case where it's not there if we ever decide to remove it in some scenarios, but since we haven't done that yet, it's not a bug

Reply:
If the test fails only for Node.js you have 2 options to fix that:
 - introduce a fake capture like you did
 - mark the test with `@bug` for current and past version. Once it is fixed with the new release, the test will be performed for the next releases. Yes the test will not be performed for the current release. is it a big deal I don't think so. It is passing currently with a capture. You will not change past releases anyway